framework/decode/vulkan_decoder_base.h

```diff
@@ -57,7 +57,7 @@ class VulkanDecoderBase : public ApiDecoder
 
     virtual void WaitIdle() override;
 
-    virtual bool IsComplete(uint64_t block_index)
+    virtual bool IsComplete(uint64_t block_index) override
     {
```

Comment:
I take it functions like this were never implemented, never called, and previous compilers / flags caused no errors.